hack/ci/e2e-k8s.sh

```diff
@@ -86,6 +86,21 @@ create_cluster() {
     esac
   fi
 
+  scheduler_extra_args="      \"v\": \"${KIND_CLUSTER_LOG_LEVEL}\""
```

Comment:
You need to pass this variable to scheduler config :P 
line 150
```
  scheduler:
    extraArgs:
        "v": "${KIND_CLUSTER_LOG_LEVEL}"
 ```

Reply:
Done , please check!